### ISSUE TITLE
Run composer install -q --no-ansi --no-interaction --no-scripts --no-progress --prefer-dist

### ISSUE THREAD

**Issue by cur1ousFranz** [LaravelDaily/Test-Laravel-Auth-Basics#182]
Hello sir Povilas, I am already in this chapter and I passed all test cases. But when I create pull request it gives me an error:

Run composer install -q --no-ansi --no-interaction --no-scripts --no-progress --prefer-dist
Your lock file does not contain a compatible set of packages. Please run composer update.
Error: Process completed with exit code 2.

What does it mean? I tried to run composer update and composer install, still the same.

**Comment by sohailahmad07** [comment]
I have that same issue too did you run the composer update locally 
I had to run composer update because I have PHP version 8.1 and the project was using PHP 8.0 
maybe I because of that

**Comment by cur1ousFranz** [comment]
Yes that's what I did too and still couldn't resolve the problem, but the most important thing is we can solve the test. I already proceed to next lessons and challenges, I really appreaciate sir povillas for making this path. Happy coding brother.
